controller/volume_controller.go

```diff
@@ -421,6 +422,38 @@ func (vc *VolumeController) getCurrentEngine(v *longhorn.Volume, es map[string]*
 	return nil, fmt.Errorf("BUG: multiple engines detected when volume %v is detached", v.Name)
 }
 
+// EvictReplicas do creating one more replica for eviction,
+// or evicting a replica. Or do nothing.
+func (vc *VolumeController) EvictReplicas(v *longhorn.Volume,
+	e *longhorn.Engine, rs map[string]*longhorn.Replica, healthyCount int) (err error) {
+	for _, replica := range rs {
+		if replica.Status.EvictionRequested == true {
+			if healthyCount == v.Spec.NumberOfReplicas {
+				if err = vc.replenishReplicas(v, e, rs); err != nil {
+					logrus.Errorf("Failed to create new replica for replica eviction err %v", err)
+					vc.eventRecorder.Eventf(v, v1.EventTypeWarning,
+						EventReasonFailedEviction,
+						"volume %v failed to create one more replica", v.Name)
+					return err
+				}
+				logrus.Debug("Creating one more replica for eviction")
+			} else {
```

Comment:
`else if healthyCount > v.Spec.NumberOfReplicas`

Reply:
This function `EvictReplicas` is in `if healthyCount >= v.Spec.NumberOfReplicas`, so inside this function, the else part should be `healthyCount > v.Spec.NumberOfReplicas`. Probably I can add a comment here.

Reply:
will fix it.